hera_cal/abscal.py

```diff
@@ -465,12 +472,14 @@ def abs_amp_logcal(self, verbose=True):
         self._abs_eta = odict(map(lambda k: (k, copy.copy(fit["eta_{}".format(k[1])])), flatten(self._gain_keys)))
         self._abs_eta_arr = np.moveaxis(map(lambda pk: map(lambda k: self._abs_eta[k], pk), self._gain_keys), 0, -1)
 
-    def TT_phs_logcal(self, verbose=True, zero_psi=False, four_pol=False):
+    def TT_phs_logcal(self, refant=None, verbose=True, zero_psi=True, four_pol=False):
```

Comment:
Why aren't you using self.refant?

Reply:
same as before, if you look `self.refant` is actually used in the function. thanks for the catch